### ISSUE TITLE
add SPA docs for ClojureScript/Reagent

### ISSUE THREAD

**Issue by yogthos** [yogthos/luminus#56]

**Comment by limist** [comment]
+1, this would be cool and useful!

**Comment by ryepesg** [comment]
:+1:

Using [re-frame](https://github.com/Day8/re-frame) or [petrol](https://github.com/krisajenkins/petrol) for elm like architecture would be nice too.

**Comment by yogthos** [comment]
yeah adding a `+re-rfame` profile probably wouldn't hurt either :)
